### ISSUE TITLE
Modifiability of the prior and posterior plots

### ISSUE THREAD

**Issue by FBartos** [bayesplay/bayesplay#10]
Could it be possible to modify the prior and posterior plots in order to modify the lines widths, types, and etc with standard ggplot2 commands?

E.g., this is the default prior & posterior plot:
```
data_mod <- likelihood(family = "normal", mean = 0, sd = 1)
h1_mod <- prior(family = "normal",    mean = 0, sd = 1)
plot(extract_posterior(data_mod * h1_mod), add_prior = TRUE)
```
![image](https://user-images.githubusercontent.com/38475991/124747975-d4fa7d00-df22-11eb-9de1-47976e1f8277.png)

I can easily change the color of the lines with
```
plot(extract_posterior(data_mod * h1_mod), add_prior = TRUE) + 
  ggplot2::scale_color_manual(values = c("black", "grey"), labels = c("posterior", "priors"))
```
![image](https://user-images.githubusercontent.com/38475991/124748107-ffe4d100-df22-11eb-9c1a-e772ff960134.png)

but I can't change the linetype
```
plot(extract_posterior(data_mod * h1_mod), add_prior = TRUE) + 
  ggplot2::scale_color_manual(values = c("black", "grey"), labels = c("posterior", "priors")) +
  ggplot2::scale_linetype_manual(values = c(1, 2), labels = c("posterior", "priors"))
```

My colleague suggested that adding a grouping variable to decide the color, linetype, and with would allow modification with something like this:
```
aes(color = group, linetype = group)
```

Btw, it's a pretty cool package and it was nice working with it! These were just a few small things that I stumbled upon. There is also no need to address any of these suggestions since I implemented way around for my own/JASP usage.

Cheers,
Frantisek

**Comment by ljcolling** [comment]
Thanks again, this is a good idea.

**Comment by ljcolling** [comment]
Fixed with #18
